lib/mail_catcher.rb

```diff
@@ -2,35 +2,32 @@
 
 # Apparently rubygems won't activate these on its own, so here we go. Let's
 # repeat the invention of Bundler all over again.
-gem "eventmachine", "1.0.9.1"
+gem 'async', '~> 1.25'
+gem 'async-http', '~> 0.56.3'
+gem 'async-io', '~> 1.32.1'
+gem 'async-websocket', '~> 0.19.0'
+gem 'falcon', '~> 0.39.1'
 gem "mail", "~> 2.3"
 gem "rack", "~> 1.5"
 gem "sinatra", "~> 1.2"
 gem "sqlite3", "~> 1.3"
-gem "thin", "~> 1.5.0"
-gem "skinny", "~> 0.2.3"
```

Comment:
I think we can remove the above gem dependency locking. This was mainly because rubygems was getting confused and activating incorrect versions, and I added this in anger. If the probably persists we can find a better solution.

Reply:
Removed this in the latest commit and atm it does not create any issue in my local